src/Connections/GenericHook.ts

```diff
@@ -148,9 +150,8 @@ export class GenericHookConnection extends BaseConnection implements IConnection
 
         return new GenericHookConnection(
             roomId,
-            state,
+            event.content,
```

Comment:
We definitely want to return the *validated* state to the constructor here, not the random crap a user might have put in the event.

Reply:
The constructor now does the validating, so that callers don't have to be mindful to pre-validate it.  But it looks like that should be reverted.

Reply:
Reverted